src/components/projects/wbs-details/work-package-container/work-package-details/work-package-details.tsx

```diff
@@ -9,7 +9,7 @@ import PageBlock from '../../../../shared/page-block/page-block';
 import { Col, Container, Row, Form } from 'react-bootstrap';
```

Comment:
idk why I can't select line 7, but Github is showing a warning that `percentPipe` isn't ever used.